scripts/verify.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Verify if B2K is disconnected successfully
```

Comment:
As far as I understand, the entire script will exit with any failure. Is there value in using something like https://github.com/torokmark/assert.sh to run the tests in a more modular way?

Reply:
@sabbour thanks for the suggestion. I will check and get back to you, I believe assert.sh are similar to exit codes.